pandas/_libs/tslibs/period.pyx

```diff
@@ -1246,6 +1246,38 @@ cdef class _Period(object):
 
     @property
     def dayofweek(self):
+        """
+        Return the day of the week.
+
+        This attribute returns the day of the week on which the particular
+        starting date for the given period occurs with Monday=0, Sunday=6.
+
+        Returns
+        -------
+        Int
+            Range of 0 to 6
```

Comment:
- I'd say correct English should be "Range from 0 to 6". So "from", not "of". But I'm not a native English speaker so please double check.
- Perhaps add (included) to make clear 6 is also included. Since `range(0, 6)` in Python doesn't include the 6. :)
- End line with period.